### ISSUE TITLE
Extension på address og internasjonale behov

### ISSUE THREAD

**Issue by ellgre** [HL7Norway/basisprofiler-r3#30]
Jeg ser at det på address har blitt lagt inn en extension for official i no-basis-address. Det er nå en diskusjon hvor blant annet Nederland og Sverige er med for å gjøre denne extension del av standard https://chat.fhir.org/#narrow/stream/194447-nordics/topic/Patient. Er det noen fra Norge som er med i diskusjonene om hvordan denne extension skal fungere?

**Comment by thomiz** [comment]
Ja, har ikke sett den, men har svart på den saken nå. Jeg ser for meg at en kode i Address.use vil fungere bedre enn en extension. Vi må isåfall skrive om mappingtabellen...

**Comment by rockphotog** [comment]
Oppfølging: Vurdere om vi skal spille inn attributt om offisiell adresse til R5. Mulig saken allerede er spilt inn. Sjekk Gforge.

**Comment by rockphotog** [comment]
Lukker R3-stuff.
